test/spec/controllers/main.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+describe('Controller: MainCtrl', function () {
+
+  beforeEach(module('todoListApp'), module('LocalStorageModule'));
+  var MainCtrl,
+    scope;
+  beforeEach(inject(function ($controller, $rootScope) {
+    scope = $rootScope.$new();
+    MainCtrl = $controller('MainCtrl', {
+      $scope: scope
+    });
+  }));
+
+  it('should have no items in the list when started', function () {
+    expect(scope.todos.length).toBe(0);
+  });
+
+  it('should add items to the list', function () {
+    scope.todo = 'Add item';
+    scope.addTodo();
+    expect(scope.todos.length).toBe(1);
+  });
+
+  it('should add an item to the list and then remove it', function () {
```

Comment:
You've already tested adding an item to the list - this test should only test removing it.